packages/fe-fpm-writer/src/view/types.ts

```diff
@@ -0,0 +1,33 @@
+import type { CustomElement, InternalCustomElement } from '../common/types';
+
+export interface CustomView extends CustomElement {
+    /**
+     * Name of the routing target.
+     */
+    target: string;
+
+    /**
+     * View title, is shown in the tab item.
+     */
+    label: string;
+
+    /**
+     * Unique tab identifier.
+     */
+    key: string;
+
+    /**
+     * If not set (i.e. undefined) then no event handler is linked. If it is set true, a new one is created and linked to the action.
```

Comment:
...set to true, a new controller is created ...

Reply:
Thanks, applied suggestion.